pages/specification/migration/_index.md

```diff
@@ -3,5 +3,85 @@ title: Migrating from older drafts
+| `dependentRequired`     | Validation    | 2019-09            | No      | Partially replaced `dependencies`                                              |
+| `dependentSchemas`      | Core          | 2019-09            | No      | Partially replaced `dependencies`                                              |
+| `deprecated`            | Validation    | 2019-09            | No      | No                                                                             |
+| `description`           | Validation    | 01                 | No      | No                                                                             |
+| `disallow`              | Core          | 01                 | 04      | Replaced by `not`                                                              |
+| `divisibleBy`           | Validation    | 02                 | Yes     | Replaced by `multipleOf`                                                       |
+| `else`                  | Core          | 07                 | No      | No                                                                             |
+| `enum`                  | Validation    | 01                 | No      | No                                                                             |
+| `examples`              | Validation    | 06                 | No      | No                                                                             |
+| `exclusiveMaximum`      | Validation    | 03                 | No      | Modified in 04                                                                |
+| `exclusiveMinimum`      | Validation    | 03                 | No      | Modified in 04                                                                |
+| `extends`               | Core          | 01                 | 04      | Replaced by `allOf`                                                            |
+| `format`                | Validation    | 01                 | No      | Modified in 04, 06, 07, 2019-09, and 2020-12                                   |
+| `id`                    | Core          | 03                 | 06      | Replaced by `$id`                                                              |
+| `if`                    | Core          | 07                 | No      | No                                                                             |
+| `items`                 | Core          | 01                 | No      | Changed in 2020-12                                                            |
+| `maxContains`           | Validation    | 2019-09            | No      | No                                                                             |
+| `maxDecimals`           | Validation    | 01                 | Yes     | Replaced by `divisibleBy`                                                      |
+| `maxProperties`         | Validation    | 04                 | No      | No                                                                             |
```

Comment:
I noticed `maxItems` and `minItems` are missing entirely in this PR. They've always existed since draft-01 and have never changed.